docs/tutorial.py

```diff
@@ -53,3 +57,41 @@
 
 # To finish off, lets plot the first z-slice of the first channel of this data:
 plt.imshow(zarr_arr[0, 0, :, :], cmap="gray")
+
+# ## Creating new datasets
+#
+# To create new OME-Zarr datasets, the ``.new()`` method on the OME-Zarr groups
+# can be used.
+#
+# As an example we'll create an OME-Zarr image with two arrays, one at the
+# original resolution and one downsampled version.
+
+arrays = [zarr.empty(shape=(100, 100)), zarr.empty(shape=(50, 50))]
+pixel_size = (6, 4)
+pixel_unit = "um"
+
+ome_zarr_image = Image.new(
+    arrays=arrays,
+    paths=["level0", "level1"],
+    axes=[
+        Axis(name="y", type="space", unit=pixel_unit),
+        Axis(name="x", type="space", unit=pixel_unit),
+    ],
+    global_scale=pixel_size,
```

Comment:
I would not include the global scale here, because it's not the strongest part of the spec and it doesn't add anything to the data model

Reply:
I use this all the time to specify the pixel size of my lowest resolution voxels, which then makes the scale transforms of each level nicer because they are then (1, 1), (2, 2), (4, 4).

Reply:
I also think we should make sure the function is flexible enough for users to be able to specify all possible fields in one shot, instead of calling this function and then having to modify metdata afterwards.

Reply:
It might make them nice integers but it also makes them inconsistent with the units defined in the `Axis` metadata. It also means the spec allows the same thing (the coordinates of an image) to be defined in two different ways, which is a bug IMO

Reply:
e.g., if I'm looking at some image metadata, it really doesn't help me to know that image X has a scale that's 2 times some other number defined elsewhere in the metadata document! I just want to know the actual scale for that image. And similarly for translation.

Reply:
> I also think we should make sure the function is flexible enough for users to be able to specify all possible fields in one shot, instead of calling this function and then having to modify metdata afterwards.

I'm fine with the global scale as a keyword argument to the method, because this method should support the full metadata model. but for this example in the docs, I would keep things simple and omit it, since it's redundant. Also, I think the global transform stuff is going away in future versions of the spec.

Reply:
I think that is a discussion for the specification, not for this package? My take is that since we're strictly following the spec, if the spec allows something then we should allow users to write it.

FWIW I don't disagree with your reasoning and would support changing the spec to remove the global scale/translation (but maybe that's going to happen anyway in the coordinates RFC?)

Reply:
> I think that is a discussion for the specification, not for this package? My take is that since we're strictly following the spec, if the spec allows something then we should provide support for users to write it.
> 
> FWIW I don't disagree with your reasoning and would support changing the spec to remove the global scale/translation (but maybe that's going to happen anyway in the coordinates RFC?)

support doesn't mean encourage. for documentation, we should show the core features users need, and not show how to support every single thing in the spec. the global scale IMO is a wart that should not be encouraged, so choosing examples judiciously is actually high-value if it steers people away from anti-patterns.

Reply:
ofc the global scale should be documented fully in API docs. but how-to guides should probably not show it, unless a bunch of users request this for some reason.

Reply:
> so choosing examples judiciously is actually high-value if it steers people away from anti-patterns.

Ah, my bad 🤦 - I thought this comment was on the function signature, not the example - I agree with removing this from example!

Reply:
Global scale removed 👍

Reply:
thank you!